app/assets/javascripts/topbar.js

```diff
@@ -1,16 +1,9 @@
 $(function() {
-  var clone = $('.persist-header').clone();
```

Comment:
No sense making two copies of the header just so once can be `static` and the other `fixed`. We can change the position css when needed.